src/js/components/Sidebar/doc.js

```diff
@@ -0,0 +1,25 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+export const doc = Sidebar => {
+  const DocumentedSidebar = describe(Sidebar)
+    .availableAt(getAvailableAtBadge('Sidebar'))
+    .description('An Sidebar.')
+    .usage(
+      `import { Sidebar } from 'grommet';
+<Sidebar/>`,
+    )
+    .intrinsicElement('div');
+
+  DocumentedSidebar.propTypes = {
+    footer: PropTypes.node.description(
+      'If specified, a footer component for the Sidebar',
```

Comment:
component => element. I React terms, think we're adding an element here, not a component.